toolkit/tools/isomaker/isomaker.go

```diff
@@ -24,6 +24,7 @@ var (
 	releaseVersion    = app.Flag("release-version", "The repository OS release version").Required().String()
 	resourcesDirPath  = app.Flag("resources", "Path to 'resources' directory").Required().ExistingDir()
 	outputDir         = app.Flag("output-dir", "Path to directory to place final image").Required().String()
+        isoGrubCfg        = app.Flag("custom-iso-grub-cfg", "Path to iso grub.cfg to place final image").String()
```

Comment:
nit: "to place in final image"
missing 'in'

Maybe also indicate that it's custom or user-provided grub.cfg (as opposed to the default one already provided by the toolkit)

Reply:
updated.